lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context_and_proc.h

```diff
@@ -268,6 +268,8 @@ typedef struct smf_context_s {
   int retransmission_count;
   protocol_configuration_options_t pco;
 
+  uint32_t count_duplicate_pdu_session_est_req;
+
```

Comment:
Can you please rename it as pdu_session_est_req_count..(or duplicate_pdu_session_est_req_count)

Reply:
Addressed review comment